_targets.R

```diff
@@ -15,76 +15,90 @@ parameterCd <- '00010'
 startDate <- "2014-05-01"
 endDate <- "2015-05-01"
 fetch_out_folder <- "1_fetch/out"
+process_out_folder <- '2_process/out'
 
 ## Fetch and save downloaded nwis data info
 p1_targets_list <- list(
   tar_target(
-    site_data_01427207,
+    site_data_01427207_csv,
```

Comment:
note: modified the name of this target as well

Reply:
Yup, looks good!